packages/flutter/lib/src/widgets/app.dart

```diff
@@ -183,7 +181,7 @@ class WidgetsApp extends StatefulWidget {
     this.title = '',
```

Comment:
The routes parameter should have a non-nullable argument type here (unlike the property)

Reply:
same with navigationObservers

Reply:
Done